architecture-decisions/pages/console.md

```diff
@@ -0,0 +1,46 @@
+ADR: Console
+============
+
+The Goose Console is a web interface that provides visual representation and control over Goose's job management features for async, scheduled, batch, periodic, and dead jobs. 
+
+Design
+-------
+
+The Goose Console is built using Hiccup for HTML templating, CSS for styling, and JavaScript for interactivity and dynamic updates.
+The console's design and mockups can be found [here](https://docs.google.com/document/d/1DmSvsdNVhsKfQ0NgxQ6ONIz0cl5_I0NhPBSmDOkwhMM/edit?usp=sharing).
+
+Web View Update
+----------------
+
+The console's web view is updated by refreshing the page to fetch the latest job data.
+If jobs are missing since the last refresh, it is likely because they have moved to a different queue (checkout job [lifecyle](../../diagrams/job-lifecycle/job-lifecycle.png))
+or have been executed by a worker.
+
+API Design
+-----------
+
+Goose does not require a dedicated server to run the console.
+Instead, the client application using Goose will have its own server, and the console functionality will be exposed via the `console/app-handler` function.
```

Comment:
> will have its own server

The wording here can be confusing, can we instead say that Goose Console runs on the same server as the other Goose components?
Can phrase the previous line (Goose does not...) and this one as a single statement.

Reply:
I find `Goose Console runs on the same server as the other Goose components` to be confusing, as this does not clarify what server we are talking about. Lets discuss this during pairing.

Reply:
Changed the lines to 
```
Goose does not provide a dedicated http server to run the console.
To serve console, the client application should call `console/app-handler` function with the appropriate `console-opts` (described below).
```